Makefile

```diff
@@ -98,6 +98,10 @@ ENVTEST_TYPE ?= kubernetes
 start-envtest: $(SETUP_ENVTEST)
 	@./hack/start-envtest.sh --environment-type=$(ENVTEST_TYPE)
 
+.PHONY: validate-dashboard
```

Comment:
Can you also call this under the `check` target?

Reply:
Also, call it `check-plutono-dashboards`. Same for the go binary name